CHANGELOG.md

```diff
@@ -5,7 +5,6 @@ Unreleased
 * [#1243](https://github.com/Shopify/shopify-cli/pull/1243): Add `tunnel` command for PHP app projects.
 * [#1245](https://github.com/Shopify/shopify-cli/pull/1245): Add `connect` command for PHP app projects.
 * [#1247](https://github.com/Shopify/shopify-cli/pull/1247): Add `open` command for PHP app projects.
```

Comment:
Wouldn't it be easier to add a single line referencing this PR that's adding support for PHP app projects, rather than have 6 lines doing so?

Reply:
I added those as I added the individual commands, let's just remove them for now and we can add them for the final PR that merges the whole thing in.